frontend/ui/widget/inputdialog.lua

```diff
@@ -142,6 +142,7 @@ local InputDialog = InputContainer:new{
                              -- needs add_nav_bar to have a Show keyboard button to get it back
     scroll_by_pan = false, -- allow scrolling by lines with Pan (= Swipe, but wait a bit at end
                            -- of gesture before releasing) (may conflict with movable)
+    use_available_height = false, -- adjust input box to fill available height on screen
```

Comment:
May be put it nearer to the one a bit related (I'd say between `cursor_at_end` and `fullscreen`), so it's easier to catch when looking for features.